modules/claims_api/app/services/claims_api/power_of_attorney_request_service/search/query.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module PowerOfAttorneyRequestService
+    module Search
+      module Query
+        module Page
+          module Size
+            # For the moment, these values duplicate the behavior of BGS.
+            DEFAULT = 25
+            MAX = 100
+            MIN = 1
+          end
+        end
+
+        module Sort
+          module Fields
+            ALL = [
+              SUBMITTED_AT = 'submittedAt'
+            ].freeze
+          end
+
+          module Orders
+            ALL = [
+              ASCENDING = 'asc',
+              DESCENDING = 'desc'
+            ].freeze
+          end
+
+          class << self
+            # These only make sense as defaults together. Also it seems sensible
+            # to not return the same hash instance when using these sort param
+            # defaults which is why this is a method.
+            def get_default
+              {
+                field: Fields::SUBMITTED_AT,
+                order: Orders::DESCENDING
+              }
+            end
+          end
+        end
+
+        # These have to load before our `Schema` definition, otherwise at least
+        # the `hints` extension won't do its thing.
+        Dry::Schema.load_extensions(:json_schema)
+        Dry::Schema.load_extensions(:hints)
```

Comment:
are we using any of these? curious if we're only using them in dev if we can bring them in conditionally?

Reply:
The `hints` extension lets us return the full set of validation failures, even those that haven't been uncovered yet by the immediate validation failure (last example on [this page of the documentation](https://dry-rb.org/gems/dry-schema/1.13/extensions/hints/)).

I to think the `json_schema` extension could only be enabled in tests.

Reply:
I think somewhat spoke to above, but again, to reiterate, `hints` is required to return a _list_ of errors? Meaning without it, as soon as it fails it reports _that spcific message_ back? And to get _all_ messages you need this extension? Schemer returns a list, so I would think moving to that would be the approach I would prefer... but glad to discuss.

Reply:
Okay I definitely want to try out `json_schemer` for the accept / deny endpoint, especially since that request payload is for a json body. I think it would be efficient to make a ticket to come back to this after I see how it works for accept / deny.

Also it gives us a little more time to debate on the question around using query params for this or not if we want. We still have a couple small pieces of nuance to settle around that I think but I'm open to whatever is the most appropriate.

For VA API standards they have commentary pertinent to search endpoints spread in I think 3 different places of the docs. I noticed the [3rd one](https://department-of-veterans-affairs.github.io/va-api-standards/paths-and-operations/paths/#collection-resources) last week. If we knew that there will not be PII in the search query, I think it would be preferable to use query params. And that brings the whole topic of supporting search parameters that BGS does not support, which is a somewhat involved question because the solution space is pretty big--they could consider relying heavily on client side searching for example.

Overall, I think we should make a ticket to revisit all of the above.